app/controllers/about/feedback_controller.rb

```diff
@@ -19,10 +19,11 @@ def create
     private
 
     def submit_feedback
-      FeedbackMailer.notify(**feedback_params.to_h
-                                             .symbolize_keys
-                                             .merge(application_env: application_environment))
-                    .deliver_later!
+      Nsm::FeedbackMailer
```

Comment:
I don't think this one should be namespaced - this is the generic feedback mailer for the feedback form that is accessible from the footer of all parts of the site, whether NSM or PA.

Reply:
good catch, the template is the same and looks generic enough to be shared at the moment so...done.